ext/hal/ti/simplelink/source/ti/devices/msp432p4xx/startup_system_files/system_msp432p401m.c

```diff
@@ -44,6 +44,7 @@
 
 #include <stdint.h>
 #include <ti/devices/msp432p4xx/inc/msp.h>
+#include <generated_dts_board.h>
```

Comment:
I am not so happy with including generated headers in ext.
Could this be done in soc.h, instead?

Reply:
that's what we do for nRF